src/main/resources/templates/db_scripts/schema.json

```diff
@@ -631,6 +631,31 @@
         {
           "fieldName": "ledgerId_fromFiscalYearId",
           "multiFieldNames": "ledgerId, fromFiscalYearId",
+          "tOps": "DELETE"
```

Comment:
If you want to delete index you can just delete it declaration from schema.json , @grabsefx please correct me if I am wrong

Reply:
@SerhiiNosko You are totally right:
![image](https://user-images.githubusercontent.com/89521577/182858586-a5472e3f-66c1-4564-8dfe-2f2e4110c511.png)

Reply:
https://github.com/folio-org/raml-module-builder#cql2pgjson-multi-field-index